Update dependencies for Laravel 7

coxy121 opened the pull request:
Add support for Laravel 7

Fixes #141

rmariuzzo commented:
I will check

On Fri, Mar 6, 2020, 19:50 Lee <notifications@github.com> wrote:

> Add support for Laravel 7
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142
> Commit Summary
>
>    - Update dependancies for Laravel 7
>
> File Changes
>
>    - *M* composer.json
>    <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780>
>    (2)
>
> Patch Links:
>
>    - https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142.patch
>    - https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142?email_source=notifications&email_token=AADLMH2MLKK3QXECLWPY6LDRGGD4RA5CNFSM4LDJ5XQKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4ITH4FQQ>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADLMH3ASNRYDEFYVDQZSHLRGGD4RANCNFSM4LDJ5XQA>
> .
>

NicolasKulka commented:
Can you validate the pull request?

alr2413 commented:
> I will check
> […](#)
> On Fri, Mar 6, 2020, 19:50 Lee ***@***.***> wrote: Add support for Laravel 7 ------------------------------ You can view, comment on, or merge this pull request online at: #142 Commit Summary - Update dependancies for Laravel 7 File Changes - *M* composer.json <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780> (2) Patch Links: - https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142.patch - https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#142?email_source=notifications&email_token=AADLMH2MLKK3QXECLWPY6LDRGGD4RA5CNFSM4LDJ5XQKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4ITH4FQQ>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADLMH3ASNRYDEFYVDQZSHLRGGD4RANCNFSM4LDJ5XQA> .

Please Merge this request if everything is ok, ASAP.
Thanks.

NicolasKulka commented:
ping @rmariuzzo

rmariuzzo commented:
The tests are not passing. I will create a separate branch for L7.

On Sun, Mar 15, 2020, 06:23 Nicolas Kulka <notifications@github.com> wrote:

> ping @rmariuzzo <https://github.com/rmariuzzo>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-599191549>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADLMHZDFY4FQTWKIZKS5KDRHSUDDANCNFSM4LDJ5XQA>
> .
>

alr2413 commented:
> The tests are not passing. I will create a separate branch for L7.
> […](#)
> On Sun, Mar 15, 2020, 06:23 Nicolas Kulka ***@***.***> wrote: ping @rmariuzzo <https://github.com/rmariuzzo> — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#142 (comment)](https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-599191549)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADLMHZDFY4FQTWKIZKS5KDRHSUDDANCNFSM4LDJ5XQA> .

When will you do that?!

rmariuzzo commented:
This weekend, I'm recovering from a flu...

On Fri, Mar 20, 2020, 04:02 @lirez@ <notifications@github.com> wrote:

> The tests are not passing. I will create a separate branch for L7.
> … <#m_-5710599438467160091_>
> On Sun, Mar 15, 2020, 06:23 Nicolas Kulka *@*.***> wrote: ping @rmariuzzo
> <https://github.com/rmariuzzo> https://github.com/rmariuzzo — You are
> receiving this because you were mentioned. Reply to this email directly,
> view it on GitHub <#142 (comment)
> <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-599191549>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AADLMHZDFY4FQTWKIZKS5KDRHSUDDANCNFSM4LDJ5XQA
> .
>
> When will you do that?!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-601576093>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADLMHYKXRRPOJECRZIITHLRIMPIFANCNFSM4LDJ5XQA>
> .
>

rmariuzzo commented:
I'm checking this now.

rmariuzzo commented:
I will start following a model similar to how https://github.com/laravel/laravel does it a “master” branch for each laravel version.

rmariuzzo commented:
Created the branch for older versions: https://github.com/rmariuzzo/Laravel-JS-Localization/tree/laravel-6.x

alr2413 commented:
> This weekend, I'm recovering from a flu...
> […](#)
> On Fri, Mar 20, 2020, 04:02 @lirez@ ***@***.***> wrote: The tests are not passing. I will create a separate branch for L7. … <#m_-5710599438467160091_> On Sun, Mar 15, 2020, 06:23 Nicolas Kulka *@*.***> wrote: ping @rmariuzzo <https://github.com/rmariuzzo> https://github.com/rmariuzzo — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <#142 (comment) <[#142 (comment)](https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-599191549)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AADLMHZDFY4FQTWKIZKS5KDRHSUDDANCNFSM4LDJ5XQA . When will you do that?! — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#142 (comment)](https://github.com/rmariuzzo/Laravel-JS-Localization/pull/142#issuecomment-601576093)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADLMHYKXRRPOJECRZIITHLRIMPIFANCNFSM4LDJ5XQA> .

We hope you start feeling better soon.
I saw you have created another branch for older versions of Laravel but you still not supported laravel 7 in master branch, is there any problem? 
Thanks

MPJHorner commented:
@rmariuzzo can I assist with anything? Be great to validate this PR, it only a 1 line fix.

rmariuzzo commented:
@MPJHorner I just need to ensure the test passes. I keep having a bad flu, not sure if 👑🦠 yet. I did some stuff locally like deleting the `tests/**/*` directories which contains configuration to test the entire projects against different Laravel version and keep `tests/specs` which contains the tests specs. However, I felt so bad that I couldn't think properly.

**Can someone create a branch that passes for Laravel 7?** Whoever takes the responsibility can remove all Laravel <7 configurations. I will greatly appreciate that work, also a group of devs will appreciate that. Due to the magnitude of the work, I can add the volunteers as co-authors, if that helps.

shadoWalker89 commented:
@rmariuzzo check the upgrade guide https://github.com/vlucas/phpdotenv/blob/master/UPGRADING.md
They did a heavy refactor, renaming name spaces, removing classes ...

coxy121 commented:
I have removed all the test directories for the old versions, and updated everything for Laravel 7, the tests are all passing and I have also tested locally pulling it in to a Laravel 7 project.

rmariuzzo commented:
Can you test https://github.com/rmariuzzo/Laravel-JS-Localization/tree/v1.6.0 ?

coxy121 commented:
I have tested the new release on a Laravel 7 project and all works as expected. Thanks for getting it updated and all your work on this.

rmariuzzo commented:
Thanks goes to @pascalbaljet.
